changelogs/unreleased/gh-3389-cfg-option-memtx-sort-threads.md

```diff
@@ -0,0 +1,4 @@
+## feature/box
+
+* Add `box.cfg` parameter `memtx_sort_threads`. It sets the number of threads
```

Comment:
Shouldn't we add a parameter, that would set `pthread_setschedprio()` or `pthread_setschedparam()` for these threads? It is actually requested by #3389